# MotorGeneralMsg.msg

component_lib/GenericComponentMsg component_property

uint8 current_limit
uint8 pwm
uint8 current
uint16 rpm

# See definitions in component_lib.h
